### DISCUSSION TITLE
Release 4.3 of license-maven-plugin

### DISCUSSION THREAD

**Discussion by hazendaz** [mathieucarbou/license-maven-plugin#638] [General & Support]
I've put all the main items into milestones of importance

https://github.com/mathieucarbou/license-maven-plugin/milestone/20?closed=1

I didn't bother with the bot library updates as probably not worth noting specifically.  Even still some of what I flagged may not be necessary.

I bumped the outstanding PR wip that isn't ready to milestone 4.4.

I think after last couple PRs ready to merge are merged, then we are ready for a release.  There are a number of good new features along with some maven adjustments that are beneficial.

**Comment by hazendaz** [comment]
@mathieucarbou See above info.  Thanks.

**Comment by mathieucarbou** [comment]
@hazendaz : thanks a lot for all your work! 
Totally OK for a new 4.3 release.
Just let me know: I don't see any outstanding Prs, if you can please confirm ?

**Reply by hazendaz** [reply]
Hi @mathieucarbou, Its good to go now.  There was one more PR from dependabot for a security issue on a test case pom, merged that.  Nothing outstanding now outside the PRs moved to 4.4.  Its otherwise good to go :)

**Comment by mathieucarbou** [comment]
release done. I had to fix an incompatibility issue in the pom concerning jacoco and the source plugin (with latest versions of both).
